src/app/dataset-create/dataset-create.component.scss

```diff
@@ -151,7 +151,13 @@ hr {
         }
     }
 
+    .switch-kind-container {
+        .description {
+            font-size: 12px;
+        }
+    }
+
     .fs-12 {
-        font-size: 12px !important;
+        font-size: 12px;
     }
```

Comment:
This is a tricky one.
There is a main problem in the wrong structure of the element.
But I assume this is out of scope here to refactor a structure :)

There is already a global class fs-12, and there is no need for the same class here,
and there will be no need in the `.switch-kind-container { ... }`.

And this is a good example of where `!important` could be the right solution :)
What I mean:
 - you can add !important to the main .fs-12 class in the styles.scss
 - and remove it here inside the component, along with the `.switch-kind-container`

Reply:
Ok, fixed